maui-toolkit/Calendar/accessibility.md

```diff
@@ -0,0 +1,309 @@
+---
+layout: post
+title: Accessibility in .NET MAUI Calendar Control | Syncfusion
```

Comment:
Update details in this
https://help.syncfusion.com/maui-toolkit/migration

Reply:
Updated It in Latest Commit